lenovo/thinkpad/t450s/default.nix

```diff
@@ -6,4 +6,15 @@
     ../../../common/pc/laptop/acpi_call.nix
     ../.
   ];
+
+  systemd.paths.trackpoint = {
+    pathConfig = {
+      PathExists = "/sys/devices/rmi4-00/rmi4-00.fn03/serio2";
```

Comment:
Could you also add a brief comment here, what this line does as you did in the commit comment?
Otherwise looks good to me.

Reply:
yeah np, i'll add that when i've got my t450s back in a working state (i've had to order a replacement touchpad for unrelated reasons)